website/docs/guides/oidc.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Common OIDC provider configurations
+---
+
+This page provides guides for configuring Weave GitOps with the most common OIDC providers.
+
+## Google
+
+1. Obtain the client ID and secret by following the [official guide](https://developers.google.com/identity/openid-connect/openid-connect)
+   from Google.
+1. Configure Weave GitOps:
+
+   ```yaml
+   apiVersion: v1
+   kind: Secret
+   metadata:
+       name: oidc-auth
+       namespace: WEAVE_GITOPS_NAMESPACE
+   stringData:
+       clientID: CLIENT_ID_FROM_STEP_1
+       clientSecret: CLIENT_SECRET_FROM_STEP_1
+       issuerURL: https://accounts.google.com
+       redirectURL: http://YOUR_WEAVE_GITOPS_DOMAIN/oauth2/callback
+       customScopes: openid,email
+   ```
```

Comment:
We should probably explain why the `customScopes` needs to be set to this, i.e. no groups data available from Google.

https://accounts.google.com/.well-known/openid-configuration doesn't support it.